src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1705,6 +1710,28 @@ public function isXmlHttpRequest()
         return 'XMLHttpRequest' == $this->headers->get('X-Requested-With');
     }
 
+    /**
+     * Checks whether the client browser prefers safe content or not according to RFC8674.
+     *
+     * @see https://tools.ietf.org/html/rfc8674
+     */
+    public function isSafePreferred(): bool
```

Comment:
the name might be confusing with `isMethodSafe`
"is safe" without context might be hard to grasp.
proposal: `preferSafeContent()`?

Reply:
Yes, that's what I thought too. :+1: for `preferSafeContent()`